postgres/init.sql

```diff
@@ -2,3 +2,4 @@ CREATE ROLE canton WITH PASSWORD 'supersafe' LOGIN;
 CREATE DATABASE mydomain OWNER canton;
 CREATE DATABASE participant1 OWNER canton;
 CREATE DATABASE jsonapi OWNER canton;
+CREATE DATABASE trigger_service OWNER canton;
```

Comment:
Maybe we can start to use the Docker compose service name as DB name:
* `http-json`
* `trigger-service`